.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,104 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: "*"
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        go_version:
+          - 1.13
+        os:
+          - ubuntu-latest
+
+    env:
+      DING_TOKEN: db178ff8c63cdc1b9ae0b86f7c4006ddf35e0249f4ae03441d80e55fdc600419
```

Comment:
Should not be exposed here, Is there any other way to set it?

Reply:
Someone with super access is needed to set up the repository's **secrets**.